conf/uppmax.config

```diff
@@ -13,6 +13,7 @@
 process {
   executor = 'slurm'
   clusterOptions = { "-A $params.project ${params.clusterOptions ?: ''}" }
+  container = "docker://$wf_container"
```

Comment:
You don#t need the docker prefix anymore. 
And you could have something like in the base profile here: https://github.com/nf-core/rnaseq/blob/2cf2f0f779d39463b09d874449cbb9fdb9f58d2f/conf/base.config#L14 
inside the profile here too - initializing the variable in nextflow.config https://github.com/nf-core/rnaseq/blob/2cf2f0f779d39463b09d874449cbb9fdb9f58d2f/nextflow.config#L15